src/randoop/operation/ConstructorSignatures.java

```diff
@@ -0,0 +1,76 @@
+package randoop.operation;
+
+import java.lang.reflect.Constructor;
+
+/**
+ * ConstructorParser provides static methods to write as well as recognize
```

Comment:
Change "recognize and extract" to "parse".
(The class creates and parses string representations of the constructor.)